2345/1/triangle.rb

```diff
@@ -0,0 +1,38 @@
+# Triangle Project Code.
+
+# Triangle analyzes the lengths of the sides of side_a triangle
+# (represented by side_a, side_b and side_c) and returns the type of triangle.
+#
+# It returns:
+#   :equilateral  if all sides are equal
+#   :isosceles    if exactly 2 sides are equal
+#   :scalene      if no sides are equal
+#
+# The tests for this method can be found in
+#   about_triangle_project.rb
+# and
+#   about_triangle_project_2.rb
+
+# :reek:UtilityFunction
+# :reek:ControlParameter
+def triangle_type(side_a, side_b, side_c)
+  if (side_a == side_b) && (side_b == side_c)
+    :equilateral
+  elsif (side_a != side_b) && (side_a != side_c) && (side_b != side_c)
+    :scalene
+  else
+    :isosceles
+  end
+end
+
+# :reek:FeatureEnvy
+def triangle(side_a, side_b, side_c)
```

Comment:
Hi. I'm a mentor from Rubizza. Alena asked to help a bit with reviews, so here am I :)

Looks good, I really dig that you moved the identification logic to other method!

Just some minor improvements for you to be aware of 🙃
Atm you validate sides _three_ times:
`(side_a >= side_b + side_c) || (side_b >= side_a + side_c) || (side_c >= side_a + side_b)`. 

I think if you sort them, then you'll be able to validate them just _once_. 
You also can _move_ that logic to another method, just as you did with `triangle_type` 😉